### ISSUE TITLE
Search bar has trailing quotation marks

### ISSUE THREAD

**Issue by coder077** [freeCodeCamp/freeCodeCamp#35431]
**Describe the bug**
When you go to the main site: learn.freecodecamp.org and type a query on the search bar, there is an odd trailing quotation mark after you hit enter.

**To Reproduce**
1. Go to learn.freecodecamp.org
2. Type in a search query on learn platform and press enter
3. A new tab opens with a trailing quotation in the query

**Expected behavior**
The query should not have any trailing quotation marks.

**Screenshots**
<img width="1017" alt="screen shot 2019-02-26 at 7 23 20 pm" src="https://user-images.githubusercontent.com/18624843/53456342-0cc09400-39fc-11e9-9977-8b83d643b3a6.png">

**Desktop (please complete the following information):**
 - OS: iOS, Ubuntu, MacOS
 - Browser: Safari

**Comment by lasjorg** [comment]
I can't reproduce it (Chrome and Firefox), what browser is this in?

**Comment by coder077** [comment]
@lasjorg Sorry Probably should have mentioned it. It appears in both Chrome and Safari but not in Firefox.

**Comment by RandellDawson** [comment]
@coder077 There have been many changes to the `master` branch which have not yet made it to production.  You can see these changes at https://freecodecamp.rocks.  If you still see these problems on the beta site, then please let us know.  If not, can you please close this issue?

Thank you.

**Comment by coder077** [comment]
@RandellDawson Yes, I checked the master branch and the error is still there. I could probably take a look at this if it's possible!

**Comment by RandellDawson** [comment]
Which error specifically?

**Comment by lasjorg** [comment]
It just redirects to https://search.freecodecamp.org when you do the search on freecodecamp.rocks

BTW. Is the search still in this archived repo?
https://github.com/freeCodeCamp/search-assets

**Comment by lasjorg** [comment]
@coder077 Can you try copy pasting something and see if you still get the trailing quotation mark.

**Comment by RandellDawson** [comment]
@lasjorg It is at the moment.  I believe there an issue which involves bringing the search functionality into the mono repo.

**Comment by lasjorg** [comment]
@RandellDawson I ask because the logo alignment is messed up on that page. Where should I post this issue? Also, would making a PR to the archived repo make any sense, I'm not sure how archived repo's work.

**Comment by coder077** [comment]
@RandellDawson When I go to learn site https://www.freecodecamp.rocks/learn and say type any query and press enter, the new tab that opens always has a trailing quotation after the search query.
@lasjorg Yes, I tried it but it is the quotation is still there.

**Comment by RandellDawson** [comment]
@coder077 This could be a Safari issue, when using Chrome with the beta site, I do not see this issue.

**Comment by lasjorg** [comment]
@coder077 So is it Chrome and Safari on iOS where it happens, but not on Ubuntu with Firefox?

Also, we are talking iOS as in the tablet/phone right, not macOS?

**Comment by coder077** [comment]
@RandellDawson Yes, turns out it is a Safari-only issue.

**Comment by coder077** [comment]
@RandellDawson @Bouncey I think I found two solutions to this problem but the changes needed to be made to the search-asset repository which is archived right now. In order to test my solution locally, I tried using lerna but ran into some errors. 
<img width="693" alt="Screen Shot 2019-04-04 at 2 10 25 PM" src="https://user-images.githubusercontent.com/18624843/55746942-111f9a00-5a09-11e9-966c-7462f6b057bc.png">

Can this be fixed or is there some better way to deal with this?
